apps/auth/main.py

```
@@ -0,0 +1,40 @@
+from config import PASSWORD_MAX_LENGTH, PASSWORD_MIN_LENGTH
+import database as db
+
+app = FastAPI(
+    title="Envybase Authentication Service",
+    description="Authentication microservice for Envybase",
+    version="0",
+)
+
+
+@app.get("/")
+def read_root():
```

Ruff (0.8.2)

Missing return type annotation for public function `read_root`

(ANN201)